LibertyDSNP/parquetjs
# Feature: Parquet Schema from JSON Schema

Problem
=======

We want it to be easier to build Parquet files (and thus the schema for those files) from existing schemas. In this case JSON Schemas

Solution
========

Change summary:
---------------
* Added field creation helper functions
* Added static function `ParquetSchema.fromJsonSchema` that takes in a Json Schema

Steps to Verify:
----------------
1. Examples are in the tests, but take a JSON schema and call `ParquetSchema.fromJsonSchema` with it and see what it does
